src/ledger/database/lmdb.zig

```diff
@@ -0,0 +1,716 @@
+const std = @import("std");
+const c = @import("lmdb");
```

Comment:
Do you prefer to use `c`? I like to use the actual name, since there could be multiple translate-c imports, and it wouldn't make sense. Just a nit.

Reply:
Interesting arguments, I agree with them.